pkg/common/jindo.go

```diff
@@ -20,15 +20,15 @@ package common
 const (
 	JindoRuntime = "jindo"
 
-	JindoChartName = "jindofs"
+	JindoChartName = "jindofsx"
```

Comment:
I suggest using  JindoChartName and JindofsxChartName to differ.

Reply:
done